main.go

```diff
@@ -47,6 +49,7 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(ramendrv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(Recipe.AddToScheme(scheme))
```

Comment:
The `KubeObjectProtectionSpec`, is in the `VRG` and `DRPC`, but it seems like the `Recipe` reference is only dereferenced by the VRG controller, so it seems like it should be inserted at line 92.